src/renderer/components/+preferences/add-helm-repo-dialog.tsx

```diff
@@ -0,0 +1,188 @@
+import "./add-helm-repo-dialog.scss"
+
+import React from "react";
+import { remote } from "electron";
+import { action, observable, runInAction } from "mobx";
+import { observer } from "mobx-react";
+import { t, Trans } from "@lingui/macro";
+import { _i18n } from "../../i18n";
+import { Dialog, DialogProps } from "../dialog";
+import { Wizard, WizardStep } from "../wizard";
+import { Input } from "../input";
+import { Checkbox } from "../checkbox";
+import { systemName, isUrl, isPath } from "../input/input_validators";
+import { SubTitle } from "../layout/sub-title";
+import { Icon } from "../icon";
+import { Notifications } from "../notifications";
+import { HelmRepo, repoManager } from "../../../main/helm/helm-repo-manager";
+
+interface Props extends Partial<DialogProps> {
+  onAddRepo: Function
+}
+
+enum FileType{
+  CaFile,
+  KeyFile,
+  CertFlie,
+}
```

Comment:
We can try to use [String enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) here to remove long switch statements in code below.
```
enum FileType {
  CaFile = "caFile",
  KeyFile = "keyFile",
  CertFile = "certFile",
}